2dr.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+    font-family: 'Roboto';
+    src: url(https://fonts.googleapis.com/css?family=Merriweather:900|Roboto)
+}
+
+@font-face{
+    font-family: 'Merriweather';
+    src: url(https://fonts.googleapis.com/css?family=Merriweather:900|Roboto)
+}
+body{
+    width: 625px;
+    color: white;
+    
+    
+}
+h1{
+    font-family: 'Merriweather';
+    font-weight: 900;
+    border-bottom: 15px;
+    border-top: 30px solid transparent;
+    margin: unset
```

Comment:
To reset a default margin value, we user `margin: 0`. unset does something similar but different.